### ISSUE TITLE
New Features Roadmap

### ISSUE THREAD

**Issue by d0p3t** [d0p3t/SubNotifier#15]
I would like to extend features on `SubNotifier` without changing the core functionality:

_Simple Subscription Notifier for Twitch Chat_

I am more than willing to expand this project. The only thing I will never support is anything outside of Twitch chat. For that, another project can be started.

Post your requests in a comment below with the following format:
<pre>
<b>Title:</b>
<b>Description:</b>
<b>Reason:</b>
<b>Expected Behaviour:</b>
</pre>

Example:
<pre>
<b>Title:</b> Add gift subscription support
<b>Description:</b> Ability to notify in chat whenever someone gifts a subscription
<b>Reason:</b> Gifting subs is a new feature on Twitch that is used a lot. 
It would be nice to thank those that are gifting subs and also notify those that 
have been granted a sub.
<b>Expected Behaviour:</b> Viewer gifts sub to another viewer. 
The bot sends a chat message to notify the chat similar to subscriptions 
and resubscriptions
</pre>

![bitmoji](https://render.bitstrips.com/v2/cpanel/71a6c3bf-709d-40f2-8992-5d9be8a0b026-cca32a10-7ddb-429b-bc89-d32f09b447ae-v1.png?transparent=1&palette=1&width=246)

**Comment by 6mas** [comment]
Title: bits/cheering 
Description:  the Ability to have a custom message for different amount of bits etc . like 1k = custom string 1  and  10k = custom string 2 , but have different then just 1k and 10k , like have a few for different amounts 
Reason: so you can write different responses to a certain amount bits dropped 
Expected Behaviour: for it  to thank people on the certain amount of bits dropped in channel , 

not sure if you  understand what im saying  about the cheering amount of certain responses

**Comment by d0p3t** [comment]
@6mas 
Thank you for your suggestion. I would like some more clarification.

Would you like to see a custom message for the TOTAL number of bits someone has cheered or in a SINGLE cheer?

**Option 1:** X cheers 200 bits to Y and now has a total of 1000 bits triggering the 1000 bits custom message
**Option 2:** X cheers 5000 bits to Y triggering the 5000 bits custom message

I looked around the [Twitch API documentation](https://dev.twitch.tv/docs/) and I could not find a way to check for total number of bits cheered by an individual (for Option 1). Perhaps it's somewhere in the user object, I'd have to check.

Option #2 is definitely possible.

**Comment by 6mas** [comment]
yes its the one you said in option 2 ,  so there's ones for certain amount cheered at once 
thanks for looking for  more features to add to the bot

**Comment by d0p3t** [comment]
Features was added in v1.0.6

It might be buggy as I did not have the resources to test this. If there are any please let me know.

You can find the new release in the Releases section or the reference above

**Comment by d0p3t** [comment]
New Features Request:

1. Prime subscriptions
2. Mass sub gifting
